tests/features/embeddables/GH3134.test.ts

```diff
@@ -0,0 +1,87 @@
+
+import {
+  Embeddable,
+  Embedded,
+  Entity,
+  MikroORM,
+  PrimaryKey,
+  Property,
+} from '@mikro-orm/core';
+
+@Embeddable()
+class Nested {
+
+  @Property()
+  field1: string;
+
+  @Property()
+  field2: string;
+
+  @Property()
+  field3: string;
+
+  constructor(field1: string, field2: string, field3: string) {
+    this.field1 = field1;
+    this.field2 = field2;
+    this.field3 = field3;
+  }
+
+}
+
+@Entity()
+class Parent {
+
+  @PrimaryKey({ autoincrement: false })
+  id: number;
+
+  @Embedded(() => Nested)
+  nested: Nested;
+
+  constructor(id: number, nested: Nested) {
+    this.id = id;
+    this.nested = nested;
+  }
+
+}
+
+let orm: MikroORM;
+
+beforeAll(async () => {
+  orm = await MikroORM.init({
+    entities: [Parent, Nested],
+    dbName: ':memory:',
+    type: 'postgresql',
```

Comment:
```suggestion
    type: 'sqlite',
```

i dont think this does what you think -- `:memory:` is a special sqlite db name, postgres does not offer such thing (but allows you to have pretty much any database name as long as its correctly quoted)

Reply:
Ah, I was careless. Thank you!